repo-index.yml

```diff
@@ -306,6 +306,25 @@ plugins:
   name: cf-download
   updated: 2016-11-01T00:00:00Z
   version: 1.2.0
+- authors:
+  - name: CF Loggregator Team
+  binaries:
+  - checksum: 8ac384dec1d0b6cbb33a70390ee757d8dba69a26
+    platform: osx
+    url: https://github.com/cloudfoundry/cf-drain-cli/releases/download/v0.6/cf-drain-cli-darwin
+  - checksum: fc28ed8d32b9d7a9f6f05686166bc9bd871b67f1
+    platform: win64
+    url: https://github.com/cloudfoundry/cf-drain-cli/releases/download/v0.6/cf-drain-cli-windows
+  - checksum: cd8703fce9a598535132de7a52ebbeae1c1ae833
+    platform: linux64
+    url: https://github.com/cloudfoundry/cf-drain-cli/releases/download/v0.6/cf-drain-cli-linux
+  company: Pivotal
+  created: 2018-04-20T00:00:00Z
+  description: A plugin to simplify interactions with user provided syslog drains.
+  homepage: https://github.com/cloudfoundry/cf-drain-cli
+  name: cf-drain-cli
```

Comment:
When install the plugin it's called `drains` but the name you're providing is `cf-drains-cli`. We're now asking for plugin developers to be consistent with the naming as we've had confusion with this in the past:
```
cf install-plugin -f https://github.com/cloudfoundry/cf-drain-cli/releases/download/v0.6/cf-drain-cli-linux
Attention: Plugins are binaries written by potentially untrusted authors.
Install and use plugins at your own risk.
Starting download of plugin binary from URL...
 8.12 MiB / 8.12 MiB [=======================================================================================================] 100.00% 3s
Installing plugin drains...
OK
Plugin drains 0.6.0 successfully installed.
```

Reply:
@XenoPhex we updated the name to be just `drains`.